packages/type-safe-api/docs/developer_guides/type-safe-api/lambda_handlers.md

```diff
@@ -440,62 +440,34 @@ When you use a handler router, you must specify the same lambda function for eve
 === "TS"
 
     ```ts
-    import { Operations } from "myapi-typescript-runtime";
-    import { Authorizers, Integrations } from "@aws-prototyping-sdk/type-safe-api";
-    import { NodejsFunction } from "aws-cdk-lib/aws-lambda-nodejs";
-    import { Construct } from "constructs";
-    import { Api } from "myapi-typescript-infra";
-    
-    export class SampleApi extends Api {
-      constructor(scope: Construct, id: string) {
-        super(scope, id, {
-          defaultAuthorizer: Authorizers.iam(),
-          // Use the same integration for every operation.
-          integrations: Operations.all({
-            integration: Integrations.lambda(new NodejsFunction(scope, "router")),
-          }),
-        });
-      }
-    }
+    new Api(this, "Api", {
```

Comment:
Should we preserve the imports so users know where it comes from?

Reply:
The imports were incorrect in the transpiled code - I'll revisit in another PR! :)